tests/api2/vmware.py

```diff
@@ -38,4 +38,3 @@ def test_02_create_vmware(data):
     results = POST('/vmware/get_datastores/', payload)
     assert results.status_code == 200, results.text
     assert isinstance(results.json(), dict) is True, results.text
-    data['vmid'] = results.json()
```

Comment:
@ericbsd can you please confirm if this change is okay wrt tests ? I could not find a usage for this. Thank you